Patrono2022

KaHea-21 opened the pull request:
# Pull Request

This PR is for a

- [x] [New Publication(s)](#new-publication)
- [ ] [Correction](#correction)

For the following list(s):

- [ ] ancientmetagenome-environmental ([README](https://github.com/SPAAM-workshop/AncientMetagenomeDir/tree/master/ancientmetagenome-environmental))
- [ ] ancientmetagenome-hostassociated ([README](https://github.com/SPAAM-workshop/AncientMetagenomeDir/tree/master/ancientmetagenome-hostassociated))
- [x] ancientsinglegenome-hostassociated ([README](https://github.com/SPAAM-workshop/AncientMetagenomeDir/tree/master/ancientsinglegenome-hostassociated))

This is to close #1329

### PR Workflow

1. Open this PR with sample metadata on the samples metadata sheet (:tada: you're already here!)
2. Wait for checks for sample metdata to pass
3. (If checks fail) make corrections, and push changes to this branch (no need to open a new PR!)
4. (Once passed) comment on this PR `@spaam-bot please autofill <table_name> <project_id>` to get a half-filled template! (may take a minute or so to get the comment with the file)
5. Fill in the template, and **verify** autofilled data correct!
6. Once filled in, append(!) the new rows from the TEMPLATE file to the end of the corresponding library metadata file, and update this PR
7. Wait for checks for library metadata to pass
8. Request review!

### Pre-review checklist (new publications)

- [x] Publication is published
  - Preprints currently not accepted?
- [x] Checked the publication is not already in the database?
- [x] Checked samples in this publication are not previously published _data_?
  - Newly re-sequenced metagenomes are OK!
- [x] Samples are shotgun metagenomes and not amplicon data
  - Note: hostassociated-singlegenome may also contain whole-genome enriched data
- [x] Checked the list follows conventions as described in the corresponding sample type's README file (e.g. using ERS/SRS accession codes for ENA/SRA)?
- [ ] **Once sample table validation completed** Library metadata has been added
  - Use `@spaam-bot please autofill <table_name> <project_id>` to get a half-filled template! May take a minute or so to get the comment with the file
  - Fill in template, and **verify** autofill data correct!
  - Once filled in, append(!) the new rows to the end of the corresponding library metadata file
- [x] Changelog is updated to include the publication under 'Added'?

KaHea-21 commented:
@spaam-bot please autofill ancientmetagenome-hostassociated Patrono2022

aidaanva commented:
@spaam-bot please autofill ancientsinglegenome-hostassociated Patrono2022

aidaanva commented:
@spaam-bot please autofill ancientsinglegenome-hostassociated Patrono2022

aidaanva commented on ancientsinglegenome-hostassociated/samples/ancientsinglegenome-hostassociated_samples.tsv:
```diff
@@ -734,3 +734,6 @@ Fiddaman2023	2023	10.1126/science.adg2238	Naderi Tepe	38.01	57.02	Iran	OL2267	Ga
 Fiddaman2023	2023	10.1126/science.adg2238	Castillo de Montsoriu	41.783	2.540	Spain	OL1999	Gallus gallus	400	10.1126/science.adg2238	virus	Gallid alphaherpesvirus 2	bone	chromosome	SRA	raw	PRJEB64489	ERS16164995
 Fiddaman2023	2023	10.1126/science.adg2238	Kazan City	55.791	49.121	Russia	OL1584	Gallus gallus	400	10.1126/science.adg2238	virus	Gallid alphaherpesvirus 2	bone	chromosome	SRA	raw	PRJEB64489	ERS16164996
 Fiddaman2023	2023	10.1126/science.adg2238	Cheboksary City	56.152	47.246	Russia	OL1585	Gallus gallus	400	10.1126/science.adg2238	virus	Gallid alphaherpesvirus 2	bone	chromosome	SRA	raw	PRJEB64489	ERS16164997
+Patrono2022	2022	10.1038/s41467-022-29614-9	Berlin	52.52	13.405	Germany	BE-572	Homo sapiens	100	10.1038/s41467-022-29614-9	virus	Influenza A virus	lung tissue	chromosome	ENA	raw	PRJEB41631	ERR5100618,ERR5100619,ERR5100620,ERR5100621
```
The ERR codes should be replaced by the ERS codes. You can access them by clicking on "Show column selection" and then the option "secondary_sample_accession".

KaHea-21 commented:
@spaam-bot please autofill ancientsinglegenome-hostassociated Patrono2022

alexhbnr commented:
It seems that the autofill struggles to find the libraries because you provide multiple sample accessions in a comma-separated list. This already is a known issue of AMDirT that we want to tackle during the hackathon. So I would suggest to keep it on hold for the moment.

jfy133 commented on ancientsinglegenome-hostassociated/samples/ancientsinglegenome-hostassociated_samples.tsv:
```diff
@@ -752,3 +752,7 @@ Seersholm2024	2024	10.1038/s41586-024-07651-2	Hunnebostrand	58.438	11.301	Sweden
 Seersholm2024	2024	10.1038/s41586-024-07651-2	Rossberga	58.230	13.606	Sweden	ROS029	Homo sapiens	5200	10.1038/s41586-024-07651-2	bacteria	Yersinia pestis	tooth	chromosome	ENA	raw	PRJEB76142	ERS19458741
 Seersholm2024	2024	10.1038/s41586-024-07651-2	Landbogarden	58.178	13.434	Sweden	CGG106001	Homo sapiens	4900	10.1038/s41586-024-07651-2	bacteria	Yersinia pestis	tooth	chromosome	ENA	raw	PRJEB76142	ERS19458685
 Seersholm2024	2024	10.1038/s41586-024-07651-2	Avlebjerg, Stroby	55.383	12.282	Denmark	AVL001	Homo sapiens	5200	10.1038/s41586-024-07651-2	bacteria	Yersinia pestis	tooth	chromosome	ENA	raw	PRJEB76142	ERS19458623
+Patrono2022	2022	10.1038/s41467-022-29614-9	Berlin	52.52	13.405	Germany	BE-572	Homo sapiens	100	10.1038/s41467-022-29614-9	virus	Influenza A virus	lung tissue	chromosome	ENA	raw	PRJEB41631	ERS5549347,ERS5549348,ERS5549349,ERS5549350
+Patrono2022	2022	10.1038/s41467-022-29614-9	Berlin	52.52	13.405	Germany	BE-576	Homo sapiens	100	10.1038/s41467-022-29614-9	virus	Influenza A virus	lung tissue	chromosome	ENA	raw	PRJEB41631	ERS5447403,ERS5549353,ERS5549354,ERS5549352,ERS5549358,ERS5549361,ERS5549351,ERS5549355,ERS5549356,ERS5549357,ERS5549359,ERS5549360
+Patrono2022	2022	10.1038/s41467-022-29614-9	Munich	48.1351	11.582	Germany	MU-162	Homo sapiens	100	10.1038/s41467-022-29614-9	virus	Influenza A virus	lung tissue	chromosome	ENA	raw	PRJEB41631	ERS6621911,ERS5447401,ERS5549344,ERS5549335,ERS5549340,ERS5549341,ERS5549343,ERS5549336,ERS5549337,ERS5549338,ERS5549339,ERS5549342
```
```suggestion
Patrono2022	2022	10.1038/s41467-022-29614-9	Berlin	52.52	13.40	Germany	BE-572	Homo sapiens	100	10.1038/s41467-022-29614-9	virus	Influenza A virus	lung tissue	chromosome	ENA	raw	PRJEB41631	ERS5549347,ERS5549348,ERS5549349,ERS5549350
Patrono2022	2022	10.1038/s41467-022-29614-9	Berlin	52.52	13.40	Germany	BE-576	Homo sapiens	100	10.1038/s41467-022-29614-9	virus	Influenza A virus	lung tissue	chromosome	ENA	raw	PRJEB41631	ERS5447403,ERS5549353,ERS5549354,ERS5549352,ERS5549358,ERS5549361,ERS5549351,ERS5549355,ERS5549356,ERS5549357,ERS5549359,ERS5549360
Patrono2022	2022	10.1038/s41467-022-29614-9	Munich	48.13	11.58	Germany	MU-162	Homo sapiens	100	10.1038/s41467-022-29614-9	virus	Influenza A virus	lung tissue	chromosome	ENA	raw	PRJEB41631	ERS6621911,ERS5447401,ERS5549344,ERS5549335,ERS5549340,ERS5549341,ERS5549343,ERS5549336,ERS5549337,ERS5549338,ERS5549339,ERS5549342
```

jfy133 reviewed (commented):
I will contonue tomorrow

jfy133 commented:
For samples:

- Missing a few ERS codes (e.g. for BE-572: ERS5447402),
- [NEED TO FINISH THIS]

For libraries:

- sample_name: should correspond to that on the `<...>samples.tsv` (e.g. `BE-152`)
- for sequencing platform: They report a mixture of `miseq` and `nextseq` in the paper, but ENA just says NextSeq for everything... @alexhbnr should we bother chasing the authors? - or we download all the submtted files and check the FASTQ headers
- for polymerase: I _think_ is 'NEBNext Ultra II Q5' based on reported library prep kit protocol here: https://www.neb.com/en/-/media/nebus/files/manuals/manuale7103-e7645.pdf?rev=de09eaf8fcdf45e0ac8a66bf6fee75fb&hash=FC51B96E7568B4482CF1B5217EF04E31

jfy133 commented:
@KaHea-21 already found a few things, see above that can be fixed while I finish :)

alexhbnr commented:
@jfy133 I will try to figure out whether there are indeed some MiSeq sequencing runs in the samples.
Regarding the polymerase, I couldn't find any real information so other than that they used the NEB kit so I left it open. If you are more certain than I was, we can replace it.

KaHea-21 commented:
@spaam-bot please autofill ancientsinglegenome-hostassociated Patrono2022

KaHea-21 commented:
@ jfy133 I’ve added the missing ERS codes to the samples and libraries tsv, and updated sample_name for libraries. I haven’t filled out all of the libraries columns for the new codes, as I’m not sure how

alexhbnr commented:
@KaHea-21 @jfy133 I have added the library data. There are two samples, ERS5447401 and ERS6621911, for which there are currently no libraries for which the number of reads in the ENA generated FastQ files is larger than 0: https://www.ebi.ac.uk/ena/browser/view/PRJEB41631

Accordingly to Seersholm 2024 (https://github.com/SPAAM-community/AncientMetagenomeDir/pull/1192#issuecomment-2760593833), we will go ahead and do not add these issues but open a new issue stating that there are samples with missing sequences that have to be added later once ENA was able to generate these FastQ files.

jfy133 commented:
> @jfy133 I am currently still checking the question whether any of the sequencing runs were sequenced on a MiSeq but otherwise I think we should be good to go regarding the libraries.

Ok then I will just let you complete and then review your own changes :grimacing: . I'll continue with Zoe's PR
